### PR TITLE
Integration e2e test #2

### PR DESCRIPTION
**Pull request by maurolacy**
Small #68 follow-up.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     uses: babylonlabs-io/.github/.github/workflows/reusable_go_lint_test.yml@v0.11.2
     with:
      run-unit-tests: true
-     run-integration-tests: true
+     run-integration-tests: false
      integration-tests-command: |
       make test-e2e
```
Comment (SebastianElvis) [inline .github/workflows/ci.yml:15]: Hmm I'd say this is still needed as `make test-e2e` uses `ibctesting` for testing everything locally. You could check the Makefile for more detail. Probably e2e is no longer an accurate word for this test 😅

File: Makefile [hunk 2]
```diff
@@ -27,10 +27,16 @@ test-all: test
 test:
 	$(MAKE) -C demo test
 	$(MAKE) -C x test
-	$(MAKE) -C tests/e2e/ test
 
-test-e2e:
+test-e2e: build-docker-e2e test-e2e-cache
+
+test-e2e-cache:
 	$(MAKE) -C tests/e2e/ test
```
Comment (SebastianElvis) [inline Makefile:34]: This is another set of e2e tests using `ibctesting`. We would still need to keep it apart from the newly added dockerised e2e tests

File: .github/workflows/ci.yml (hunk 1 above)
Comment (maurolacy) [reply .github/workflows/ci.yml:15]: These tests are empty AFAIK.

File: Makefile (hunk 2 above)
Comment (maurolacy) [reply Makefile:34]: Let's rename these tests then (if they are not void).

**Pull request merged**